docs/contribute/api-docs-contribution/guidelines.md

```diff
@@ -0,0 +1,575 @@
+# Add core content
+
+This page covers the core guidelines for excellent API docs. Learn how to write clear summaries and descriptions, create helpful examples, and add useful links.
+
+## Write summaries and descriptions
+
+Most objects in your Open API specification (i.e. operations, parameters, properties, etc.) accept both concise summaries and detailed descriptions to help users understand their purpose and usage.
+
+### Write summaries
+
+Clear, single-sentence summaries help users understand the components of your API at a glance. They appear in various contexts like IDEs, search results, and documentation overviews.
+
+Here are some principles for writing effective summaries:
+
+- **Be concise:** Keep summaries short (between 5-45 characters) because they appear in different contexts where space is limited
+- **Start with a verb:** Use action words like "Get", "Update", "Delete", "Create"
+- **Use simple verbs:** Use simple verbs (Get, Update, Delete) rather than verbose alternatives (Retrieve, Return, List)
+- **Include articles:** "Delete a space", "Delete spaces", "Delete all spaces"
+- **Use sentence case:** Capitalize only the first word and proper nouns
+
+#### Summary examples
+
+::::{tab-set}
+:group: implementations
+
+:::{tab-item} OpenAPI
+:sync: openapi
+
+In OpenAPI specifications, summaries are defined in the `summary` field:
+
+```yaml
+paths:
+  /indices/{index}:
+    get:
+      summary: Get index information
+      description: Retrieves configuration and mapping information...
```

Comment:
In the wiki we currently recommend the less passive form (though obviously haven't cleaned it up in all the APIs yet)

```suggestion
      description: Retrieve configuration and mapping information...
```